Makefile

```diff
@@ -123,7 +123,11 @@ ifeq ($(UNAME_S),Darwin)
 			-I$(DEP_DIR)compatibility/filesystem \
 			-include "base/macos_allocator_replacement.hpp"
     LIBS += -framework CoreFoundation
-    SHARED_ARGS += -mmacosx-version-min=10.14 -arch x86_64 -D_LIBCPP_STD_VER=16
+    SHARED_ARGS += \
+      -mmacosx-version-min=10.12 \
```

Comment:
I am confused by the use of spaces in f12b910a49d67f9f0055fe0dbc23132e9c29e037, I thought Makefiles needed tabs?
The surrounding code seems to use a weird mixture of spaces for indentation and tabs for continuation lines so I am not sure what to think.